hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -56,7 +56,7 @@
   }
 
   private static Iterable<Object[]> useJdbcAndSchemaFromCommitMetadata() {
-    return Arrays.asList(new Object[][] { { true, true }, { true, false }, { false, true }, { false, false } });
+    return Arrays.asList(new Object[][] {{true, true}, {true, false}, {false, true}, {false, false}});
```

Comment:
Same here and below

Reply:
@n3nash : same comment as above. Can you go to this file in one of your branch and apply reformatting and let me know if you see similar changes. If not, I will looking into my reformatting is diff.